torch_geometric/explain/explainer.py

```diff
@@ -203,3 +203,54 @@ def _threshold(self, explanation: Explanation) -> Explanation:
             explanation[key] = mask
 
         return explanation
+
+    def visualize_feature_importance(
+        self,
+        explanation: Explanation,
+        feat_labels: Optional[List[str]] = None,
+        n_features: int = 20,
+    ):
+        r"""Creates a bar plot of the node features importance given
+        a :attr:`node_feat_mask`
+
+        Args:
+            node_feat_mask (Tensor): The node feature mask to plot.
+            feat_labels (List[str], optional): Optional labels for features.
+            n_features (int, optional): Top N features to plot.
+                (default: :obj:`20`)
```

Comment:
Update doc:
1. Use `explanation` instead of `node_feat_mask`.
2. rename `n_features` to `top_k` for consistency.